docker/.env.example

```
@@ -15,3 +15,12 @@ OPTIMIZE_DEFAULT_PRETTIFY=true
 CLAIMED_PERFORMANCE_SCORE=
 CLAIMED_PERFORMANCE_APP_VERSION=
 CLAIMED_PERFORMANCE_MEASURED_AT=
+USAGE_METRICS_ENABLED=true
+USAGE_METRICS_RETENTION_DAYS=30
+USAGE_REPORT_LOCAL_HOUR=0
+USAGE_REPORT_TRANSPORT=stdout
+MAILGUN_API_URL=https://api.mailgun.net/v3
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MAILGUN_API_URL key should go before the OPTIMIZE_DEFAULT_PRETTIFY key

(UnorderedKey)